tests/core/test_planning.py

```diff
@@ -0,0 +1,81 @@
+# Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import numpy as np
+import pytest
+
+import mbrl.planning as planning
+
+SEED = 0
+
+
+def create_pid_agent(dim,
+                     state_mapping=None,
+                     batch_dim=1):
+    agent = planning.PIDAgent(k_p=np.random.randn(dim, ),
+                              k_i=np.random.randn(dim, ),
+                              k_d=np.random.randn(dim, ),
+                              target=np.zeros(dim, ),
+                              state_mapping=state_mapping,
+                              batch_dim=batch_dim,
+                              )
+    return agent
+
+
+def test_pid_agent_one_dim():
+    """
+    This test covers the creation of PID agents in the most basic form.
+    """
+    np.random.seed(SEED)
+    pid = create_pid_agent(dim=1)
+    pid.reset()
+    init_obs = np.random.randn(1)
+    act = pid.act(init_obs)
+
+    # check action computation
+    assert act == pytest.approx(-7.043, 0.1)
```

Comment:
Is this true regardless of the value of `init_obs`? Or is it hard coded for this seed?